src/components/Actions/ActionCopyNtoFN.vue

```diff
@@ -0,0 +1,51 @@
+<!--
+  - @copyright Copyright (c) 2018 John Molakvoæ <skjnldsv@protonmail.com>
+  -
+  - @author John Molakvoæ <skjnldsv@protonmail.com>
+  -
+  - @license GNU AGPL version 3 or any later version
+  -
+  - This program is free software: you can redistribute it and/or modify
+  - it under the terms of the GNU Affero General Public License as
+  - published by the Free Software Foundation, either version 3 of the
+  - License, or (at your option) any later version.
+  -
+  - This program is distributed in the hope that it will be useful,
+  - but WITHOUT ANY WARRANTY; without even the implied warranty of
+  - MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+  - GNU Affero General Public License for more details.
+  -
+  - You should have received a copy of the GNU Affero General Public License
+  - along with this program. If not, see <http://www.gnu.org/licenses/>.
+  -
+  -->
+
+<template>
+	<ActionButton icon="icon-up" @click="copyNtoFN">
+		{{ t('contacts', 'Copy to full name') }}
+	</ActionButton>
+</template>
+<script>
+import { ActionButton } from 'nextcloud-vue'
+import ActionsMixin from 'Mixins/ActionsMixin'
+
+export default {
+	name: 'ActionCopyNtoFN',
+	components: {
+		ActionButton
+	},
+	mixins: [ActionsMixin],
+	methods: {
+		copyNtoFN() {
```

Comment:
```suggestion
		copyNToFN() {
```

https://stackoverflow.com/questions/15526107/acronyms-in-camelcase